app/shared/fonts/Inter.ts

```diff
@@ -0,0 +1,8 @@
+import { Inter } from 'next/font/google'
+
+// it's better to keep reusable font in separate module.
+// See https://nextjs.org/docs/pages/building-your-application/optimizing/fonts#reusing-fonts
+export const interFont = Inter({
```

Comment:
Maybe keep all fonts in one file?

Reply:
no, some pages might need different fonts, we don't want to load unused ones for nothing